src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/SwitchControllerV2.java

```diff
@@ -169,4 +170,16 @@
     public CompletableFuture<SwitchConnectionsResponse> getConnections(@PathVariable("switch_id") SwitchId switchId) {
         return switchService.getSwitchConnections(switchId);
     }
+
+    /**
+     * Get switch properties.
+     *
+     * @return switch ports description.
+     */
+    @ApiOperation(value = "Get switch properties from the switch", response = SwitchPropertiesDto.class)
+    @GetMapping(value = "/properties")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<List<SwitchPropertiesDto>> getSwitchProperties() {
```

Comment:
I propose to wrap respose list into some object, so it will become: 
```json
{ 
"switch_properties": [
  {"..": "..."},
  {"..": "..."}
]
}
```

Reply:
Done